### PR TITLE
#410: added duration to latest commit for a remote branch to mob status

### LINKED ISSUE #410: mob status should show if a mob session exists
It would be helpful when `mob status` shows if a mob session exists on the current branch, when not on a mob branch, to look up if a remote mob branch already exists.

Something like this:

    ➜  myProject git:(main) mob status
    > you are on base branch 'main'
    > wip branch 'mob/main' contains 7 commits. The last 5 were:
    > 38209edb 6 hours ago <Wanda Hess>
    > 2359e605 5 hours ago <Ross Carter>
    > 3868b5c6 5 hours ago <Claudia Osborne>
    > b848355a 5 hours ago <Lee Klein>
    > 209e8bb2 5 hours ago <Ammar Carlson>

### PR DESCRIPTION
**Pull request by hollesse**
closes #410 

the result of the change can be seen here. 
```
$ mob status
> you are on base branch 'main'
> remote wip branches detected:
  - origin/mob/main (10 months ago)
  - origin/mob/main-my-feature (1 year, 4 months ago)

```

if there is no remote wip branch:
```
$ mob status
> you are on base branch 'main'
> no remote wip branches detected!
```

### CODE REVIEW

File: status_test.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	}
+
+	status(configuration)
+	// 6 wip commits + 1 start commit
+	assertOutputContains(t, output, "wip branch 'mob-session' contains 7 commits.")
+}
+
+func TestStatusDetectsWipBranches(t *testing.T) {
+	output, configuration := setup(t)
+	start(configuration)
```
Comment (hollesse) [inline status_test.go:43]: I would like to adjust the test that it does not depend on the empty commit when doing mob start. As I would like to remove this empty commit again.

Comment (gregorriegler) [reply status_test.go:43]: ok, great!

File: status_test.go [hunk 2]
```diff
@@ -41,7 +42,10 @@ func TestStatusWithMoreThan5LinesOfLog(t *testing.T) {
 func TestStatusDetectsWipBranches(t *testing.T) {
 	output, configuration := setup(t)
 	start(configuration)
+	createFile(t, "test.txt", "contentIrrelevant")
+	next(configuration)
 	git("checkout", "master")
+	time.Sleep(2 * time.Second)
```
Comment (gregorriegler) [inline status_test.go:48]: ouch, do we really need that?

Comment (hollesse) [reply status_test.go:48]: I think so. On my machine i ran into the problem that it was exactly `one second ago` and the test failed. One other option would be to just assert `ago` or `second` what do you think?

**Pull request merged**